pkg/webreg/webreg.go

```diff
@@ -1455,7 +1455,7 @@ const credentialExample = `ref:
   credentials:
   - namespace: test-credentials # this entry injects the custom credential
     name: my-data
-    mountPath: /var/run/my-data
+    mount_path: /var/run/my-data
```

Comment:
Just a vote of support. This is correct. make validate-step-registr throws invalid error on the mountPath field, but accepts mount_path